sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/AsyncErrorMappingTest.groovy

```diff
@@ -0,0 +1,158 @@
+package com.azure.storage.file.datalake
+
+import com.azure.storage.file.datalake.models.DataLakeStorageException
+import reactor.test.StepVerifier
+
+class AsyncErrorMappingTest extends APISpec {
+
+    DataLakeFileSystemAsyncClient fsac
+    String fileSystemName
+
+    def setup() {
+        fileSystemName = generateFileSystemName()
+        fsac = getServiceAsyncClient(primaryCredential).createFileSystem(fileSystemName).block()
+    }
+
+    def "Read file"() {
+        when:
+        def fileName = generatePathName()
+        def fac = fsac.getFileAsyncClient(fileName)
+        def readFileVerifier = StepVerifier.create(fac.readWithResponse(null, null, null, false))
+        then:
+        readFileVerifier.verifyErrorSatisfies {
```

Comment:
I think there's an overload of verifyError that just accepts the type of the error. Then you don't have to write these lambdas

Reply:
Yep, changed to use that